bot/sx-bot.el

```diff
@@ -0,0 +1,82 @@
+;;; sx-bot.el --- Functions for viewing different tabs.       -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2014  Artur Malabarba
+
+;; Author: Artur Malabarba <bruce.connor.am@gmail.com>
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+;;; Commentary:
+
+;;
+
+
+;;; Code:
+
+(require 'sx-site)
+(require 'sx-tag)
+
+(setq sx-request-remaining-api-requests-message-threshold 50000)
+
+(require 'package)
+(package-initialize)
```

Comment:
It was required to be run from the command line.  Things like `let-alist` and `json` weren't defined – things we very much need for the tests. I think we should require `sx-load` though – what do you think?